package.json

```diff
@@ -1,134 +1,135 @@
+		"expo-device": "~4.0.3",
+		"expo-document-picker": "~10.0.3",
+		"expo-font": "~10.0.3",
+		"expo-image-picker": "~11.0.3",
+		"expo-linear-gradient": "~10.0.3",
+		"expo-linking": "~2.4.2",
+		"expo-localization": "~11.0.0",
+		"expo-location": "~13.0.4",
+		"expo-notifications": "~0.13.3",
+		"expo-permissions": "~13.0.3",
+		"expo-screen-orientation": "~4.0.3",
+		"expo-secure-store": "~11.0.3",
+		"expo-splash-screen": "~0.13.5",
+		"expo-updates": "~0.10.15",
+		"graphql": "14.7.0",
+		"graphql-anywhere": "~4.2.4",
+		"graphql-tag": "^2.12.6",
+		"lodash": "~4.17.21",
+		"markdown-it": "^12.3.2",
+		"matomo-tracker-react-native": "^0.3.0",
+		"moment": "~2.29.1",
+		"qs": "^6.10.3",
+		"react": "17.0.1",
+		"react-apollo": "~3.1.5",
+		"react-hook-form": "^6.11.0",
+		"react-native": "^0.64.3",
+		"react-native-app-intro-slider": "^4.0.4",
+		"react-native-autocomplete-input": "^4.2.0",
+		"react-native-calendars": "^1.1276.0",
+		"react-native-collapsible": "^1.6.0",
+		"react-native-elements": "~1.2.0",
+		"react-native-expo-image-cache": "4.1.0",
+		"react-native-gesture-handler": "~1.10.2",
+		"react-native-get-random-values": "~1.7.0",
+		"react-native-gifted-chat": "0.16.3",
+		"react-native-markdown-display": "^7.0.0-alpha.2",
+		"react-native-modal-dropdown": "https://github.com/donni106/react-native-modal-dropdown#feature/search-options",
+		"react-native-qrcode-svg": "^6.1.1",
+		"react-native-reanimated": "~2.2.0",
+		"react-native-render-html": "4.2.5",
+		"react-native-safe-area-context": "3.3.2",
+		"react-native-screens": "~3.8.0",
+		"react-native-snap-carousel": "~3.9.0",
+		"react-native-svg": "12.1.1",
+		"react-native-webview": "11.13.0",
+		"react-native-webview-leaflet": "https://github.com/okwast/react-native-webview-leaflet#d1914b1b02dc7a9faad45f23abadbf1fd3958abc",
+		"react-query": "^3.34.14",
+		"sentry-expo": "^4.0.0",
+		"styled-components": "~4.4.1"
+	},
+	"devDependencies": {
+		"@babel/core": "^7.12.9",
+		"@babel/eslint-parser": "^7.17.0",
+		"@babel/plugin-proposal-logical-assignment-operators": "^7.12.1",
+		"@babel/preset-typescript": "^7.13.0",
+		"@types/lodash": "^4.14.165",
+		"@types/react": "~17.0.21",
+		"@types/react-dom": "~17.0.9",
+		"@types/react-native": "~0.64.12",
+		"@types/react-native-calendars": "^1.505.2",
+		"@typescript-eslint/eslint-plugin": "^4.8.2",
+		"@typescript-eslint/parser": "^4.8.2",
+		"babel-jest": "25.5.1",
+		"babel-preset-expo": "8.5.1",
+		"eslint": "^7.6.0",
+		"eslint-config-prettier": "^7.1.0",
+		"eslint-plugin-import": "^2.23.4",
+		"eslint-plugin-prettier": "^3.3.0",
+		"eslint-plugin-react": "7.20.6",
+		"eslint-plugin-react-native": "3.9.1",
+		"eslint-utils": "1.4.3",
+		"jest": "25.5.4",
+		"jest-expo": "^43.0.0",
+		"jest-styled-components": "^7.0.3",
+		"mixin-deep": "^1.3.2",
+		"prettier": "^2.2.1",
+		"react-test-renderer": "^17.0.2",
+		"set-value": "^4.1.0",
+		"typescript": "~4.3.5"
+	},
+	"resolutions": {
+		"markdown-it": "^12.3.2"
```

Comment:
this file is linted wrongly again, i will fix this before merge to see the changes